webviz_config/plugins/_markdown.py

```diff
@@ -81,21 +81,25 @@ def handleMatch(self, m, data: str) -> tuple:  # type: ignore[no-untyped-def]
 
 
 class Markdown(WebvizPluginABC):
-    """### Include Markdown
+    """Renders and includes the content from a Markdown file.
 
-_Note:_ The markdown syntax for images has been extended to support
-(optionally) providing width and/or height for individual images.
-To specify the dimensions write e.g.
-```markdown
-![width=40%,height=300px](./example_banner.png "Some caption")
-```
+---
 
-Renders and includes the content from a Markdown file. Images are supported,
-and should in the markdown file be given as either relative paths to
-the markdown file itself, or absolute paths.
+* **`markdown_file`:** Path to the markdown file to render and include. \
+                       Either absolute path or relative to the configuration file.
+
+---
+
+Images are supported, and should in the markdown file be given as either
+relative paths to the markdown file itself, or as absolute paths.
+
+> The markdown syntax for images has been extended to support (optionally) \
+  providing width and/or height for individual images. \
```

Comment:
Easier to read when not breaking the sentence with parentheses.
```suggestion
> The markdown syntax for images has been extended to support \
  providing width and/or height for individual images (optional). \
```